cfgmgr/coppmgr.cpp

```diff
@@ -687,7 +683,7 @@ void CoppMgr::doCoppTrapTask(Consumer &consumer)
         }
         else if (op == DEL_COMMAND)
         {
-            if (conf_present && !m_coppTrapConfMap[key].trap_group.empty() && !m_coppTrapConfMap[key].trap_ids.empty())
+            if (conf_present)
```

Comment:
Can you please explain why this check is removed?

Reply:
it is here https://github.com/Azure/sonic-swss/pull/2034/files#diff-39e9e71a7788fdd27fda2f3bd818b47ce232b447bb3d1064b1f1433e51657459R704